app/services/database_manager.py

```diff
@@ -0,0 +1,231 @@
+from qdrant_client import QdrantClient
+from qdrant_client.http import models as rest
+from sentence_transformers import SentenceTransformer
+import fitz
+
+# Configuration for Qdrant client
+client = QdrantClient(host="localhost", port=6333)
+
+# Sentence-BERT model
+model = SentenceTransformer('sentence-transformers/all-MiniLM-L6-v2')
+
+
+def partition_text(text, chunk_size=100):
+    """
+    Partitions text into chunks of specified size.
+
+    Parameters:
+    text (str): The text to be partitioned.
+    chunk_size (int): The size of each text chunk.
+
+    Returns:
+    list: A list of text chunks.
+    """
+    words = text.split()
+    chunks = [' '.join(words[i:i + chunk_size]) for i in range(0, len(words), chunk_size)]
+    return chunks
+
+
+def generate_embeddings(text_chunks):
+    """
+    Generates embeddings for each text chunk using Sentence-BERT.
+
+    Parameters:
+    text_chunks (list): A list of text chunks.
+
+    Returns:
+    list: A list of embeddings.
+    """
+    embeddings = model.encode(text_chunks)
+    return embeddings
+
+
+def create_collection(collection_name):
+    """
```

Comment:
Be careful with docstrings and comments. They will create a lot of maintenaiblity issues in the long term. In this case, if you use proper variable typing `collection_name: str` the purpose of the functions is completely clear and basically you will not require any docstring. I do recommend docstrings if the code is going to be delivered as a framework or library to final users or if the parameters are not 100% clear on its functioning.